plugins/dashboard/frontend/src/app/components/TangleTime.tsx

```diff
@@ -16,19 +16,54 @@ export default class TangleTime extends React.Component<Props, any> {
         return (
             <Card>
                 <Card.Body>
-                    <Card.Title>TangleTime Synced: {this.props.nodeStore.status.tangleTime.synced? "Yes":"No"} </Card.Title>
+                    <Card.Title>TangleTime
+                        Synced: {this.props.nodeStore.status.tangleTime.synced ? "Yes" : "No"} </Card.Title>
                     <small>
                         <div>
                             <hr/>
-                            <div>Message: <Link to={`/explorer/message/${this.props.nodeStore.status.tangleTime.messageID}`}>
-                                            {this.props.nodeStore.status.tangleTime.messageID}
-                                        </Link></div>
-                            <div>Time: {dateformat(new Date(this.props.nodeStore.status.tangleTime.time/1000000), "dd.mm.yyyy HH:MM:ss")}</div>
+                            <div className={"row"}>
```

Comment:
This is nice but it "jumps" around at least in my browser. Maybe we can make the font for the message a bit smaller or something? Or simply put the second one in the next row

<img width="1138" alt="image" src="https://user-images.githubusercontent.com/4181434/174309980-f618ebff-843f-42ee-9e4c-4811475db370.png">